src/main/java/io/cryostat/net/web/http/api/v2/graph/StartRecordingOnTargetMutator.java

```diff
@@ -138,7 +138,8 @@ public HyperlinkedSerializableRecordingDescriptor getAuthenticated(
                                     .create(conn.getService())
                                     .name((String) settings.get("name"));
                     if (settings.containsKey("restart")) {
-                        restart = Boolean.parseBoolean((String) settings.get("restart"));
+                        Boolean v = (Boolean) settings.get("restart");
```

Comment:
I think the more Java-y way would be `boolean v = Boolean.TRUE.equals(settings.get("restart"))`, but that's just stylistic.